backend/src/api/user-change-password.js

```diff
@@ -2,16 +2,16 @@ import bcrypt from "bcryptjs";
 import User from "../models/user.js";
 
 const changePassword = async (req, res) => {
-  const { email, oldPassword, newPassword } = req.body;
+  const { id, oldPassword, newPassword } = req.body;
```

Comment:
While email technically should be a unique identifier id is better as it's uniqueness is guaranteed